lib/models/installation.js

```diff
@@ -57,6 +58,43 @@ module.exports = (sequelize, DataTypes) => {
     async uninstall(payload) {
       return Installation.destroy({ where: { ownerId: payload.account.id } });
     },
+
+    async assertPermissions(github, { owner, repo, permissions }) {
```

Comment:
@bkeepers adding this function here already which is in may ways similar to #587  but different in the sense that permissions can be asserted outside of the middleware. Curious for your thoughts

Reply:
I like having it in the model, and the command middleware can just reuse this.